src/pages/home/report/ReportActionsList.tsx

```diff
@@ -209,9 +209,24 @@ function ReportActionsList({
             ),
         [sortedReportActions, isOffline],
     );
+
+    const newestVisibleReportAction = useMemo(() => {
+        // whisper action doesn't affect lastVisibleActionCreated, so we should not take it into account while checking if there is the newest report action
+        let index = 0;
+        const size = sortedVisibleReportActions.length;
+        while (index < size) {
```

Comment:
Isn't a `for (let index = 0; i < sortedVisibleReportActions.length; i++)` more suited for this? or even further

```
sortedVisibleReportActions.find(ReportActionsUtils.isWhisperAction) ?? null
```
?

Reply:
good idea with using `find` - code simplified